static/src/javascripts-legacy/projects/common/modules/onward/tech-feedback.js

```diff
@@ -110,23 +51,89 @@ define([
         }
     }
 
-    /**
-     * the link in the footer adds some of the values to the hash so feedback can use it later.  Those values
-     * override those at the time the email is sent.
-     */
+    function toggleFormVisibility() {
+        $.forEachElement("#feedback-category>option", function(elem){
```

Comment:
Rather than hardcode `#feedback-category` in here, it's probably be better to get it off the passed `event object (`event.target`). You can get the value of the selected option using `event.target.value`:

```js
document.getElementById("feedback-category").addEventListener("change", toggle, false);

function toggle(event) {
  console.log(event.target.value)
}
```